src/pasta/fields/fq.rs

```diff
@@ -661,43 +638,43 @@ impl ff::PrimeField for Fq {
 impl FieldExt for Fq {
     const ROOT_OF_UNITY: Self = ROOT_OF_UNITY;
     const ROOT_OF_UNITY_INV: Self = Fq::from_raw([
-        0x824860d3eb30de02,
-        0xad9f0afd0ea63acc,
-        0xd250318c11a16fe1,
-        0x12a9f5e1dd62dabc,
+        0x57eecda0a84b6836,
+        0x4ad38b9084b8a80c,
+        0xf4c8f353124086c1,
+        0x2235e1a7415bf936,
     ]);
     const UNROLL_T_EXPONENT: [u64; 4] = [
-        0x9b71de17e6d2d5a0,
-        0x0000000000296ee0,
-        0x8c00000000000000,
-        0x2ecc05e,
+        0xcc771cc2ac1e1664,
+        0x00000000062dfe9e,
+        0xc000000000000000,
+        0xb89e9c7,
     ];
     const T_EXPONENT: [u64; 4] = [
-        0xda58a1b2610b2bf5,
-        0x0000000000e2a849,
+        0x0994a8dd8c46eb21,
+        0x00000000224698fc,
         0x0000000000000000,
-        0x10000000,
+        0x40000000,
     ];
     const DELTA: Self = DELTA;
-    const UNROLL_S_EXPONENT: u64 = 0x344cfe85d;
+    const UNROLL_S_EXPONENT: u64 = 0xd1d858e1;
```

Comment:
I did not check this constant, and I can't find any unit test for it.

Reply:
See https://github.com/zcash/halo2/pull/84#discussion_r541156767

Reply:
Couldn't check this; please document.

Reply:
Checked.